models/collection.js

```diff
@@ -3,8 +3,12 @@
 const Joi = require('joi');
 const mutate = require('../lib/mutate');
 
+const BUILD_MODEL = require('./build').get;
 const PIPELINE_MODEL = require('./pipeline').get;
-const PIPELINES_MODEL = Joi.array().items(PIPELINE_MODEL);
+const PIPELINE_OBJECT = PIPELINE_MODEL.keys({
+    lastBuilds: Joi.array().items(BUILD_MODEL).optional()
```

Comment:
I discussed it with @d2lam and @jithin1987 earlier and we concluded that it would make more sense to show all the builds for the last event rather than just the very last build for the event to make it easier to understand where it fails etc.